### PR TITLE
[MDCT-2732] - Add basic structure for StandardReportPages

### PR DESCRIPTION
**Pull request by ntsummers1**
### Description

Adds the StandardReportPage and its children to the project! 
Screenshot 2023-08-08 at 4 45 03 PM

### Related ticket(s)

MDCT-2732

---
### How to test

If you add 
```
<Route
    path="/standard"
    element={<StandardReportPage route={mockStandardReportPageJson} />}
/>
``` 
to AppRoutes.tsx, you can view a very simplistic view by changing the route to /standard

---
### Author checklist

- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [x] I have updated relevant documentation, if necessary

### CODE REVIEW

**Review by britt-mo — APPROVED** [review]
looked as expected in browser per `how to test`

File: services/ui-src/src/types/reports.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { CustomHtmlElement, FormJson } from "types";
+
+export interface ReportPageVerbiage {
+  intro: {
+    section: string;
+    subsection?: string;
+    spreadsheet?: string;
```
Comment (karla-vm) [inline services/ui-src/src/types/reports.ts:7]: I don't think we'll need the Spreadsheet icon for MFP, but we'll double-check with R&D!

File: services/ui-src/src/utils/testing/mockForm.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+export const mockForm = {
+  id: "mock-form-id",
+  fields: [mockFormField],
+};
+
+export const mockVerbiageIntro = {
+  section: "mock section",
+  subsection: "mock subsection",
+  spreadsheet: "mock item",
```
Comment (karla-vm) [inline services/ui-src/src/utils/testing/mockForm.tsx:18]: Same as above.

**Review by karla-vm — DISMISSED** [review]
Looks awesome, thanks Nick! 💯

**Review by gmrabian — APPROVED** [review]
looks good locally!

File: services/ui-src/src/utils/testing/mockForm.tsx (hunk 2 above)
Comment (ntsummers1) [reply services/ui-src/src/utils/testing/mockForm.tsx:18]: Removed!

**Review by karla-vm — APPROVED** [review]

**Pull request merged**